modules/fastcv/include/opencv2/fastcv/mser.hpp

```diff
@@ -15,107 +15,98 @@ namespace fastcv {
- *
- * @param src Source image of type CV_8UC1. Image width has to be greater than 50, and image height has to be greater than 5.
-              Pixels at the image boundary are not processed. If boundary pixels are important
-              for a particular application, please consider padding the input image with dummy
-              pixels of one pixel wide.
- * @param contours Array containing found contours
- * @param boundingBoxes Array containing bounding boxes of found contours
- * @param numNeighbors Number of neighbors in contours, can be 4 or 8
- * @param delta Delta to be used in MSER algorithm (the difference in grayscale values
-                within which the region is stable ).
-                Typical value range [0.8 8], typical value 2
- * @param minArea Minimum area (number of pixels) of a mser contour.
-                Typical value range [10 50], typical value 30
- * @param maxArea Maximum area (number of pixels) of a  mser contour.
-                Typical value 14400 or 0.25*width*height
- * @param maxVariation Maximum variation in grayscale between 2 levels allowed.
-                Typical value range [0.1 1.0], typical value 0.15
- * @param minDiversity Minimum diversity in grayscale between 2 levels allowed.
-                Typical value range [0.1 1.0], typical value 0.2
- */
-CV_EXPORTS void MSER(InputArray src, std::vector<std::vector<Point>>& contours, std::vector<cv::Rect>& boundingBoxes,
-                       unsigned int numNeighbors = 4,
-                       unsigned int delta = 2,
-                       unsigned int minArea = 30,
-                       unsigned int maxArea = 14400,
-                       float        maxVariation = 0.15f,
-                       float        minDiversity = 0.2f);
+    /**
+     * @brief Creates MSER detector
+     *
+     * @param imgSize Image size. Image width has to be greater than 50, and image height has to be greater than 5.
+     * @param numNeighbors Number of neighbors in contours, can be 4 or 8
+     * @param delta Delta to be used in MSER algorithm (the difference in grayscale values
+                    within which the region is stable ).
+                    Typical value range [0.8 8], typical value 2
+     * @param minArea Minimum area (number of pixels) of a mser contour.
+                      Typical value range [10 50], typical value 30
+     * @param maxArea Maximum area (number of pixels) of a  mser contour.
+                      Typical value 14400 or 0.25*width*height
+     * @param maxVariation Maximum variation in grayscale between 2 levels allowed.
+                           Typical value range [0.1 1.0], typical value 0.15
+     * @param minDiversity Minimum diversity in grayscale between 2 levels allowed.
+                           Typical value range [0.1 1.0], typical value 0.2
+     * @return Feature detector object ready for detection
+     */
+    CV_WRAP static Ptr<MSER> create(cv::Size     imgSize,
```

Comment:
I recommend to change the interface as follows:
```
CV_WRAP static Ptr<fastcv::MSER> create(const cv::Size& imgSize,
```
Otherwise it leads to Java binding build issue (cv::MSER and cv::fastcv::MSER conflict).